examples.md

```diff
@@ -295,6 +296,30 @@ Esy allows you to export built dependencies and import pre-built dependencies.
       ${{ runner.os }}-composer-
 ```
 
+## PowerShell
+
+For PowerShell, the cache directory will vary by OS.
+
+Locations:
+ - Ubuntu: `~/.local/share/powershell/Modules`
+ - Windows: `~\Documents\PowerShell\Modules`
+ - macOS: `~/.local/share/powershell/Modules`
+
+```yaml
+- name: Get PowerShell Cache Directory
+  id: powershell-cache
+- uses: actions/cache@v2
+  with:
+    path: "~/.local/share/powershell/Modules"
+    key: ${{ runner.os }}-PS-SqlServer-PSScriptAnalyzer
```

Comment:
Could we add a comment to make it clear that the cache key here should match the modules listed in the `Install-Module` step?

Reply:
@joshmgross Is this a needed step?

Reply:
Back when I made this, I didn't entirely understand keys. I've since created a [GitHub Action](https://github.com/marketplace/actions/powershell-module-cache) to help others cache but I'd still like to keep this example. I believe this would potentially be a good key.

```
key: ${{ runner.os }}-${{ hashFiles("~/.cache/powershell/ModuleAnalysisCache-*") }}
```

Going to test it out and come back to update with the info

Reply:
`ModuleAnalysisCache` takes at least 10 seconds to be built and doesn't exist in time to be reliability tested. I'm still searching for an ideal solution.

Reply:
@potatoqualitee Are you still working on this?